platform/authorized_api_guide.md

```diff
@@ -38,22 +73,247 @@ curl 'https://<domain>/auth/token' -H "Content-Type: application/json" --data '{
+Response example:
 ```
 {"token_type":"Bearer","expires_in":86400,"access_token":"your_access_token"}
 ```
 
-Piwik PRO API tokens use [jwt](https://jwt.io/) format.
+Field `access_token` contains your token which then will be used for all API calls.
 
-Now, you can use obtained `access_token` for communication with Piwik PRO API.
-Field `expires_in` stands for time (in seconds) for token TTL.
-As token is a Bearer type, it must be **included in all API calls** within header.
+#### Create an app
+
+Request example:
 
 ```
-Authorization: Bearer your-token-here
+POST /api/apps/v2
 ```
-  
-## Deleting API Credentials
 
-Once you want to revoke the possibility of generating API token using given `CLIENT ID` and `CLIENT SECRET`,
-go to to `My profile` and click `Delete` button on selected API credentials.
+```
+curl 'https://<domain>/api/apps/v2'  -H "Authorization: Bearer <your_access_token>" -H "Content-Type: application/vnd.api+json" --data '{
+  "data": {
+    "attributes": {
+      "timezone": "UTC",
+      "name": "AppName",
+      "urls": [
+        "http://example.com"
+      ],
+      "currency": "USD"
+    },
+    "type": "ppms/app"
+  }
+}'
+
+```
+
+Note, that you have to replace: 
+* `<domain>` with your PPMS instance URL,
+* `<your_access_token>` with your generated access token 
+
+Response example:
+```
+{
+   "data":{
+      "type":"ppms\/app",
+      "id":"b30e538d-4b05-4a75-ae25-7eb565901f38",
+      "attributes":{
+         "name":"AppName",
+         "addedAt":"2018-09-13T12:16:30+00:00",
+         "urls":[
+            "http:\/\/example.com"
+         ],
+         "timezone":"UTC",
+         "currency":"USD",
+         "excludeUnknownUrls":false,
+         "keepUrlFragment":true,
+         "eCommerceTracking":false,
+         "siteSearchTracking":true,
+         "siteSearchQueryParams":[
+            "q",
+            "query",
+            "s",
+            "search",
+            "searchword",
+            "keyword"
+         ],
+         "siteSearchCategoryParams":[
+
+         ],
+         "delay":500,
+         "excludedIps":[
+
+         ],
+         "excludedUrlParams":[
+
+         ],
+         "excludedUserAgents":[
+
+         ],
+         "gdpr":true,
+         "gdprUserModeEnabled":false,
+         "privacyCookieDomainsEnabled":false,
+         "privacyCookieExpirationPeriod":31536000,
+         "privacyCookieDomains":[
+
+         ],
+         "organization":"default",
+         "appType":"web",
+         "gdprLocationRecognition":false
+      }
+   }
+}
+
+```
+#### Read an app
+
+Now, when app is added, it is possible to read it.
+
+Request example:
+```
+GET /api/apps/v2/{appId}
+```
+
+Request example:
+```
+curl 'https://testing.piwik.pro/api/apps/v2/b30e538d-4b05-4a75-ae25-7eb565901f38'  -H "Authorization: Bearer <your_access_token>" -H "Content-Type: application/vnd.api+json"
+```
+
+Notice:
+* URL contains `b30e538d-4b05-4a75-ae25-7eb565901f38`. What is it? It is unique ID of an app. 
```

Comment:
maybe it would be nice to wrap these notices in `>`? so it would be formatted like:
> Notice:
> 
> URL contains `b30e538d-4b05-4a75-ae25-7eb565901f38`. What is it? It is unique ID of an app.